src/common/components/hosts/HostStatus.tsx

```diff
@@ -138,6 +138,8 @@ const HostStatusPopoverContent: React.FC<HostStatusPopoverContentProps> = (props
           {statusDetails}
           <br />
           {toSentence(statusInfo)}
+          <br />
+          {statusInfo.includes('ISO') && 'Please generate a new minimal ISO and reboot the hosts.'}
```

Comment:
A source of the problem is failing host validation. If the validationInfo message is not broad enough the UI codebase has HOST_VALIDATION_FAILURE_HINTS which lets us add additional description to the failing validation.